### PR TITLE
Karbon Services Rebrand

### PR DESCRIPTION
**Pull request by sharvil-kekre**
- Didn't change instance of _Xi IoT Sensor App_

- Didn’t change instances of _Nutanix .Next 2019 Xi IoT HOL_

### CODE REVIEW

File: cli/samples/datapipelines/readme.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Creating a Data Pipeline
 
-Read about data pipeline details and requirements in the [Xi IoT Admin Guide](https://portal.nutanix.com/#/page/docs/details?targetId=Xi-IoT-Infra-Admin-Guide:Xi-IoT-Infra-Admin-Guide).
+Read about data pipeline details and requirements in the [Karbon Platform Services for IoT Admin Guide](https://portal.nutanix.com/#/page/docs/details?targetId=Xi-IoT-Infra-Admin-Guide:Xi-IoT-Infra-Admin-Guide).
```
Comment (piased100) [inline cli/samples/datapipelines/readme.md:3]: these links will change once KPS docs are active.

File: cli/README.md [hunk 2]
```diff
@@ -1,100 +1,100 @@
...
 **32-bit**
-[xi-iot-v1.0.0-rc5-darwin_386.tar.gz](https://xi-iot-cli.s3-us-west-2.amazonaws.com/xi-iot-v1.0.0-rc5-darwin_386.tar.gz)
+[kpsctl-v1.0.0-rc5-darwin_386.tar.gz](https://kpsctl-cli.s3-us-west-2.amazonaws.com/kpsctl-v1.0.0-rc5-darwin_386.tar.gz)
 
 Optionally, from your terminal or shell, use `wget`.
 ```
-$ wget https://xi-iot-cli.s3-us-west-2.amazonaws.com/xi-iot-v1.0.0-rc5-darwin_amd64.tar.gz
-$ mkdir xi-iot && tar zxvf xi-iot-v1.0.0-rc5-darwin_amd64.tar.gz -C xi-iot
-$ sudo xi-iot/install [*optional_path*]
+$ wget https://kpsctl-cli.s3-us-west-2.amazonaws.com/kpsctl-v1.0.0-rc5-darwin_amd64.tar.gz
```
Comment (piased100) [inline cli/README.md:26]: These links are giving 403 forbidden, assume we need to change permissions

File: cli/samples/applications/readme.md [hunk 3]
```diff
@@ -26,12 +26,12 @@ appYamlPath: echoserver-k8s.yaml
...
 | description |     | Describe your application |
 | project  |  | Specify an existing project by project name to associate with this resource. If project uses an explicit edge association, specify the edges also. See the next sample YAML file for reference |
-| appYamlPath |  | Specify the k8s yaml file path for your app. The path could be absolute or relative to the location of Xi IoT YAML file. |
+| appYamlPath |  | Specify the k8s yaml file path for your app. The path could be absolute or relative to the location of Karbon Platform Services for IoT YAML file. |
 
 ## echo-ifc-app.yaml
 
 ``` yaml
-# Note: This sample requires a Xi IoT cloud instance or deployments. See https://www.nutanix.com/products/iot
+# Note: This sample requires a Karbon Platform Services for IoT cloud instance or deployments. See https://www.nutanix.com/products/iot
```
Comment (piased100) [inline cli/samples/applications/readme.md:34]: @akhileshjoshi11 
I assume product/iot will persist right ?

File: cli/samples/datapipelines/readme.md [hunk 4]
```diff
@@ -98,11 +98,11 @@ output:
 |  | type | Here, `Azure`. |
 |  | service | Here, `Blob` indicates Azure blob storage |
 |  | endpointName | Here, name of the Azure blob |
-|  | profile | Specify the name of your cloud profile. The [Xi IoT Admin Guide](https://portal.nutanix.com/#/page/docs/details?targetId=Xi-IoT-Infra-Admin-Guide:Xi-IoT-Infra-Admin-Guide) describes how to create your cloud profile. |
+|  | profile | Specify the name of your cloud profile. The [Karbon Platform Services for IoT Admin Guide](https://portal.nutanix.com/#/page/docs/details?targetId=Xi-IoT-Infra-Admin-Guide:Xi-IoT-Infra-Admin-Guide) describes how to create your cloud profile. |
```
Comment (piased100) [inline cli/samples/datapipelines/readme.md:101]: link will change

File: cli/samples/datapipelines/readme.md (hunk 1 above)
Comment (sharvil-kekre) [reply cli/samples/datapipelines/readme.md:3]: Is this the correct link address to change to: https://portal.nutanix.com/page/documents/details?targetId=Karbon-Platform-Services-Admin-Guide:Karbon-Platform-Services-Admin-Guide?

File: cli/README.md (hunk 2 above)
Comment (sharvil-kekre) [reply cli/README.md:26]: No bucket permissions are fine, but I need to change version in link to rc7: wget https://kpsctl-cli.s3-us-west-2.amazonaws.com/kpsctl-v1.0.0-rc7-darwin_amd64.tar.gz

File: cli/samples/datapipelines/readme.md (hunk 1 above)
Comment (piased100) [reply cli/samples/datapipelines/readme.md:3]: yeah seems correct

Comment (sharvil-kekre) [reply cli/samples/datapipelines/readme.md]: updated

File: cli/samples/datapipelines/readme.md (hunk 4 above)
Comment (sharvil-kekre) [reply cli/samples/datapipelines/readme.md]: updated

File: cli/README.md (hunk 2 above)
Comment (sharvil-kekre) [reply cli/README.md]: updated to rc8

**Pull request merged**